CHANGELOG.md

```diff
@@ -7,12 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Please refer to the [SDK docs](https://www.datocms.com/docs/building-plugins/sdk-reference) for furthe information about APIs.
 
+## [0.1.1] - 2021-03-16
+
+### Added
+
+- `loadItemTypeFields(itemTypeId)` - Plugins can use this method to load all fields belonging to a specific ItemType, including relative modular blocks fields.
```

Comment:
@souljuse "Plugins can use this method to load all fields belonging to a specific model or block"

Reply:
E non hai aggiunto `environment`.